Battleship/Fleet.cs

```diff
@@ -8,11 +8,11 @@ public Fleet ()
         {
             Ships = new List<Ship>
             {
-                new Ship(ShipClass.Destroyer),
-                new Ship(ShipClass.Submarine),
-                new Ship(ShipClass.Cruiser),
-                new Ship(ShipClass.Battleship),
-                new Ship(ShipClass.Carrier)
+                new Ship(ShipClass.Destroyer, Direction.South, new Coordinate(0,0)),
```

Comment:
Zero-based coordinates, or does 0 indicate unassigned? Either is good, just curious.

Reply:
0 based coordinates. But what I need to do is figure out a way to set the direction to "unassigned" because when the fleet is instantiated they are all stacked on top of each other. 
I thought maybe an optional parameter but I couldn't remember how to do it so I opted for the simple setup of having them stacked ontop of each other on the 0,0 coordinate.